index.js

```diff
@@ -49,32 +49,31 @@ exports.register = function(plugin, options, next) {
     }
 
     agenda.define(name, opts, method.bind(plugin));
-  });
 
-  if(options.every) {
```

Comment:
Mistake... Originally I was under the thinking that batch jobs are not created as recurring jobs so there would be no need.